### PR TITLE
Week1 - Modified index.html, script.js and style.css files

### PR DESCRIPTION
**Pull request by sebagabs**
Modified `index.html`, `script.js` and `style.css` according to the Week 1 Walkthrough. Plus, created the `/images` directory inside `/webapp` and added a `.png` image. I have not completed the walkthrough so there will be another (or more) pull request(s). I send this one ahead because of our meeting, where you showed me how to do this.

### CODE REVIEW

**Comment by sebagabs** [comment]
Further modified the `index.html`, `style.css`, and `script.js` files, plus added more images in `/images`.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,18 +2,55 @@
...
+              <a href="http://www.linkedin.com/in/sgmr"> <img src="images/linkedicon.png" alt="LinekdIn" class="icons"></a>
+            </div>
+      <div class="column">
+          <a href="https://resume.creddle.io/resume/73t8u7pfoz4"> <img src="images/resumeicon.png" alt="Resume" class="icons2"></a>
+        </div>
+        <div class="column">
+            <a href="http://github.com/sebagabs"> <img src="images/githubicon.png" alt="GitHub" class="icons"></a>
+        </div>
+    </div>
+      <p>My name is Sebastián Maldoando, and I was born and raised on the island of Puerto Rico. Currently I'm a
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/index.html:23]: For practice, try adding a hyperlink to Puerto Rico on Google Maps!

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,18 +2,55 @@
...
+          be able to find a virutal portfolio containing different details about me. This webpage was developed as part of participating in 
+          Google's Software Product Sprint Spring 2021.
+      </p>
+      <a href="https://www.discoverpuertorico.com/"> <img src="images/pr-flag.png" alt="Puerto Rico flag icon." class="center"/></a>
+      <p> I was born in May of '01 in the town of Arecibo, but lived most of my life in Hatillo. For two years I lived in Corozal, and for almost 4 years now I have lived
+          in the northern town of Manatí. I enjoy eating; I love trying new foods and getting to know cultures through their gastronmic delights. That being said, I truly
+          believe that my island has one of the best varieties of meals and plates! One of my passions is to take pictures of the sky. I feel the sky is so underrated, so I
+          try to capture it's magic as best as I can. Personally, it's the night sky that has the most beauty to it; a gateway to the universe. So, I could say I do "photogrpahy"
+          for fun with my handy smartphone camera.
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/index.html:33]: Nit: Try to keep lines <= 100 chars.I would not refactor this entire document and would leave as is, but use for reference for the future!

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -17,7 +17,9 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['I did not learn how to ride a bike until 13.', 'My favorite color is red.', 'Puerto Rico is the oldest colony in the world at the moment, and I am one of its residents.', 
+    'I have a mild obsession with instant ramen with boiled eggs.', 'My favorite saga is Star Wars.', 'I used to practice Taekwondo until I had kneee surgery.', 
+    'I have 4 siblings; 2 brothers and 2 sisters.', 'I am a professional procrastinator!'];
```
Comment (GoogleyGabriel) [inline portfolio/src/main/webapp/script.js:22]: Try to keep max line length at 100 chars for JS files. Add new lines where necessary to meet this requirement.

**Review by GoogleyGabriel — APPROVED** [review]
Looking great, Sebastian!

**Pull request merged**